### PR TITLE
Overhaul quickstarts

### PR DESCRIPTION
**Pull request by thomasheremans**
1. Separation of Leader/Creator flow vs Operator flow
2. Review and clarifying copy changes to all other quickstarts

### CODE REVIEW

File: docs/int/quickstart/quickstart-cli.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+  - If the Charon cluster is running on a different chain, **ALL** operators must update `--epoch` to the same latest fork version returned by `curl $BEACON_NODE/eth/v1/config/fork_schedule`.
+- Run the command to submit this node's partially signed voluntary exit:
+  - `docker-compose -f compose-voluntary-exit.yml up`
+  - Confirm the logs: `Exit for validator XXXXX submitted`
+  - Exit the container: `Ctrl-C`
+- The charon metric `core_parsigdb_exit_total` will be incremented each time a voluntary exit partial signature is received, either from this node or from peers.
+
+## Feedback
+
+If you have gotten this far through the process, and whether you succeeded or failed at running the distributed validator successfully, we would like to hear your feedback on the process and where you encountered difficulties. Please let us know by joining and posting on our [Discord](https://discord.gg/obol). Also, feel free to add issues to our [GitHub repos](https://github.com/ObolNetwork).
```
Comment (OisinKyne) [inline docs/int/quickstart/quickstart-cli.md:202]: Same again with discord link

File: docs/intro.md [hunk 2]
```diff
@@ -17,4 +17,4 @@ This manual is aimed at developers and stakers looking to utilize the Obol Netwo
 
 ## Need assistance?
 
-If you have any questions about this documentation or are experiencing technical problems with any Obol-related projects, head on over to our [Discord](https://discord.gg/n6ebKsX46w) where a member of our team or the community will be happy to assist you.
+If you have any questions about this documentation or are experiencing technical problems with any Obol-related projects, head on over to our [Discord](https://discord.gg/obol) where a member of our team or the community will be happy to assist you.
```
Comment (OisinKyne) [inline docs/intro.md:20]:
```suggestion
If you have any questions about this documentation or are experiencing technical problems with any Obol-related projects, head on over to our [Discord](https://discord.gg/n6ebKsX46w) where a member of our team or the community will be happy to assist you.
```

File: docusaurus.config.js [hunk 3]
```diff
@@ -107,11 +107,11 @@ const config = {
             items: [
               {
                 label: 'Discord',
-                href: 'https://discord.gg/n6ebKsX46w',
+                to: 'https://discord.gg/obol',
               },
               {
                 label: 'Twitter',
-                href: 'https://twitter.com/obolnetwork',
+                to: 'https://twitter.com/obolnetwork',
```
Comment (OisinKyne) [inline docusaurus.config.js:114]: Why is href changing to `to` here? Also same change on the discord link

**Review by OisinKyne — APPROVED** [review]

**Review by OisinKyne — APPROVED** [review]
Also, please Control+F `https://launchpad.obol.dev` and replace with `https://goerli.launchpad.obol.tech`

File: docs/int/quickstart/quickstart-alone.md [hunk 4]
```diff
@@ -21,99 +27,97 @@ Ensure you have [docker](https://docs.docker.com/engine/install/) and [git](http
...
+```sh
+# Start the distributed validator cluster
+docker-compose up --build
+```
+Check the monitoring dashboard and see if things look all right
 
 ```sh
-docker run --rm -v "$(pwd):/opt/charon" obolnetwork/charon:v0.12.0 create cluster --withdrawal-address="0x000000000000000000000000000000000000dead" --nodes 6 --threshold 5
+# Open Grafana
+open http://localhost:3000/d/laEp8vupp
```
[the remark this answers is not in the document]
Comment (thomasheremans) [reply docs/int/quickstart/quickstart-alone.md:76]: I just tried and seems to work, unless it opens the wrong dashboard?

File: docusaurus.config.js (hunk 3 above)
Comment (thomasheremans) [reply docusaurus.config.js:114]: basically removes the arrow in the footer, as some links had it while others didn't

**Pull request merged**